packages/runtime/src/builder/serialization/controls/style-v2.ts

```diff
@@ -0,0 +1,24 @@
+import { StyleV2ControlDefinition } from '../../../controls'
+import { Serialize, Deserialize } from './types'
+
+export function serializeStyleV2Control(
+  control: StyleV2ControlDefinition,
+): [Serialize<StyleV2ControlDefinition>, Transferable[]] {
+  const { getStyle, ...config } = control.config
+
+  const transferables: Transferable[] = []
+  return [
+    {
+      ...control,
```

Comment:
You need to recursively serialize the control here, otherwise this will throw an error for controls that have values that can't be serialized by `postMessage`